src/show.rs

```diff
@@ -117,10 +117,18 @@ pub fn show_index_json(band: &Band, w: &mut dyn Write) -> Result<()> {
         .map_err(|source| Error::SerializeIndex { source })
 }
 
-pub fn show_entry_names<E: Entry, I: Iterator<Item = E>>(it: I, w: &mut dyn Write) -> Result<()> {
+pub fn show_entry_names<E: Entry, I: Iterator<Item = E>>(
+    it: I,
+    w: &mut dyn Write,
+    long_listing: bool,
+) -> Result<()> {
     let mut bw = BufWriter::new(w);
     for entry in it {
-        writeln!(bw, "{}", entry.apath())?;
+        if long_listing {
+            writeln!(bw, "{} {} {}", entry.umode(), entry.owner(), entry.apath())?;
```

Comment:
It seems like this will get messy if the owners vary from one line to another since it's not doing any column alignment...

Reply:
I'm not sure how best to address this. I agree it could get messy but I'm not sure it's worth doing a proper column alignment.

Reply:
In a sense the best approach, and what's probably done by table-drawing crates, is to find the maximum length for each column and then use that to set the width. However that would mean reading the whole index into memory before printing anything, which goes against a general rule for Conserve of avoiding ever needing to have the whole index in memory.

So pragmatically I think just setting them to a fixed width of 10 for the mode and 10(??) for the user and group would be fine.

(I think modern Unix has a long limit on usernames, maybe 32 characters, which would make this unreasonably wide. But under 10 seems very common?)

So just `{:<10}` (https://doc.rust-lang.org/std/fmt/#fillalignment)

Reply:
Ok cool, that's an easy change.